### ISSUE TITLE
Restore option to hide the visual radar (show only columns)

### ISSUE THREAD

**Issue by tclift** [AOEpeople/aoe_technology_radar#432]
The original PR #74 to add the visual radar came with a `config.json` option `homepageContent`, which could be set to `columns`, `chart` or `both`. This option was removed somewhere along the way.

The original reason that the AOE Technology Radar appealed to me was the lack of a visual radar, instead focusing on the functional content. I find the visual radar distracting and a form of "mystery meat navigation".

Can the `homepageContent` option please be restored, so that we can remove the visual radar again?

**Comment by mathiasschopmans** [comment]
Good morning @tclift,
Sorry for the inconvenience. I will restore this option.

**Comment by mathiasschopmans** [comment]
This will be reintroduced with the upcoming release, which will be published probably tomorrow. If you want to test it now, you can use the latest build directly via your `package.json` and run `npm update`

```json
  "dependencies": {
    "aoe_technology_radar": "github:AOEpeople/aoe_technology_radar#main"
  }
```


To create the desired output, you'll need to add update and configure the toggles by adding the following block to your `config.json`

```
"toggles": {
    "showChart": false,
    "showTagFilter": false,
    "showQuadrantList": true,
    "showEmptyRings": false
  },
```

Same feature will close issue #431 too.

**Comment by mathiasschopmans** [comment]
Feature has been released with v4.1.0
